sklearn/naive_bayes.py

```diff
@@ -523,6 +523,11 @@ def _check_X_y(self, X, y, reset=True):
         return self._validate_data(X, y, accept_sparse="csr", reset=reset)
 
     def _update_class_log_prior(self, class_prior=None):
+        """Update class log priors.
+
+        The class log priors are based on `class_prior`, class count or the
+        number of classes. This method is called each time `fit` or
+        `partial_fit` update the model.
```

Comment:
To make this a docstring:

```suggestion
        `partial_fit` update the model.
        """
```

Reply:
Yes, thanks, I was just fixing it, but got stuck.